.gitignore

```diff
@@ -1 +1,4 @@
 node_modules
+.DS_Store
+dist/app.css
```

Comment:
I totally agree that normally generated files like this should not be checked into the repo. However, to make it easy to deploy on the media wiki server without having to install dependencies and build in on the server, I think we need to include it here.
```suggestion
```